### DISCUSSION TITLE
Make the Tier 4 mythic vendor items available at the T6 mythic vendor

### DISCUSSION THREAD

**Discussion by Da1sgaard** [chromiecraft/chromiecraft#7495] [Suggestions]
I think putting the T4 vendor items at the T6 vendor would be nice QoL, or being able to buy T4 coins with T6 coins. Could even be a 2-1 or 5-1 ratio, that doesn't matter.
Basically how it works with emblems in woltk, you can downgrade emblems, but not upgrade.

**Comment by Belodina** [comment]
This is an interesting idea, perhaps unnecessary for immediate addition to the game. But converting mythic points "down" for older raid content is cool!

**Comment by TotallyNotMyAccount** [comment]
Realistically, is anyone ever going to do this while TBC is current? And I'm pretty sure mythic content is going the way of the dodo once lvl 80 wrath comes, at the latest. Seems like a waste of dev time for very minor gain tbqh.
